### PR TITLE
MAINT-27699 : Auto activities of documents

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**
Hide activities (in comments forms when renaming, moving, or adding tags to a document)

### CODE REVIEW

**Review by ahamdi — CHANGES_REQUESTED** [review]
I think there is a misunderstanding with the requirement, we need to disable the generation of those comments when the Documents activities are disabled (check documentation here https://docs.exoplatform.org/en/6.0/Configuration.html?highlight=activity%20disable#enabling-disabling-any-activity-type ) and not to disable them completely.

**Review by ahamdi — DISMISSED** [review]
LGTM
Please propose the needed content to add to the documentation of Activity types about the activity types events of rename document, move document etc ..

**Comment by mzorai** [comment]
> LGTM
> Please propose the needed content to add to the documentation of Activity types about the activity types events of rename document, move document etc ..

@Rdidmohamedelhadi yep please add a task into the board with the needed details, thanks

**Comment by Rdidmohamedelhadi** [comment]
> > LGTM
> > Please propose the needed content to add to the documentation of Activity types about the activity types events of rename document, move document etc ..
> 
> @Rdidmohamedelhadi yep please add a task into the board with the needed details, thanks

ok

File: ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/listener/Utils.java [hunk 1]
```diff
@@ -96,7 +96,12 @@
   private static int    MAX_SUMMARY_LINES_COUNT    = 4;
   private static int    MAX_SUMMARY_CHAR_COUNT     = 430;
   private static String activityType;
+  private static final String CREATION_MESSAGE = "files:spaces.CREATION_COMMENT";
+  private static final String RENAME_COMMENT   = "files:spaces.RENAME_COMMENT";
+  private static final String MOVE_COMMENT  = "files:spaces.MOVE_COMMENT";
   private static final String RESOURCE_BUNDLE_KEY_CREATED_BY = "SocialIntegration.messages.createdBy";
+  private static final String RESOURCE_BUNDLE_KEY_EDIT_NAME   = "SocialIntegration.messages.editName";
```
Comment (ahamdi) [inline ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/listener/Utils.java:103]:
```suggestion
  private static final String RESOURCE_BUNDLE_KEY_RENAME   = "SocialIntegration.messages.reame";
```

File: ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/listener/Utils.java [hunk 2]
```diff
@@ -432,14 +437,20 @@ public static ExoSocialActivity postFileActivity(Node node, String activityMsgBu
...
-    }
     ActivityManager activityManager = CommonsUtils.getService(ActivityManager.class);
     activityType = StringUtils.isNotEmpty(activityType) ? activityType : FILE_SPACES;
     if(! activityManager.isActivityTypeEnabled(activityType)) {
       return null;
     }
+      if (RESOURCE_BUNDLE_KEY_CREATED_BY.equals(activityMsgBundleKey)  && !isSystemComment && ! activityManager.isActivityTypeEnabled(CREATION_MESSAGE)) {
+          return null;
+      }
+      if (RESOURCE_BUNDLE_KEY_EDIT_NAME.equals(activityMsgBundleKey) && ! activityManager.isActivityTypeEnabled(RENAME_COMMENT)) {
```
Comment (ahamdi) [inline ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/listener/Utils.java:448]:
```suggestion
      if (RESOURCE_BUNDLE_KEY_FILE_RENAMED.equals(activityMsgBundleKey) && ! activityManager.isActivityTypeEnabled(RENAME_COMMENT)) {
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**